UnityProject/Assets/Ardity/Scripts/Threads/AbstractSerialThread.cs

```diff
@@ -44,7 +44,7 @@ public abstract class AbstractSerialThread
 
     // Internal synchronized queues used to send and receive messages from the
     // serial device. They serve as the point of communication between the
-    // Unity thread and the Ardity thread.
+    // Unity thread and the SerialComm thread.
```

Comment:
This is a change that should not be in the PR either.

Reply:
Inverted